### DISCUSSION TITLE
Help! plz tell me Your OS

### DISCUSSION THREAD

**Discussion by shogo1229** [NAIST-SE/AttractingContributors#940] [General]
plz tell me your OS
あなたが使用しているPCのOSを教えてください！

https://github.com/shogo1229/AttractingContributors

**Comment by bconfessor** [comment]
Done! Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md

**Comment by k-mizoguch1** [comment]
plz https://github.com/kzu000/Food-what-do-you-like
